src/containerapp/azext_containerapp/_params.py

```diff
@@ -323,6 +323,7 @@ def load_arguments(self, _):
         c.argument('min_executions', type=int, help="Minimum number of job executions to run per polling interval.")
         c.argument('max_executions', type=int, help="Maximum number of job executions to run per polling interval.")
         c.argument('polling_interval', type=int, help="Interval to check each event source in seconds. Defaults to 30s.")
+        c.argument('scale_rule_name', options_list=['--scale-rule-name', '--srn'], help="The name of the scale rule.")
```

Comment:
You add this new param in `_params.py` is not enough.
You need to update the function [create_containerappsjob](https://github.com/Azure/azure-cli-extensions/blob/f77c214919b322c063e98d2d0df2a7fe4e22f318/src/containerapp/azext_containerapp/custom.py#L906)

Besides, please add test for the new params.